src/modules/top-level.nix

```diff
@@ -250,6 +261,8 @@ in
       pkgs.pkg-config
     ];
 
+    hardeningDisable = [];
```

Comment:
This is already handled with the default

Reply:
Fixed